imap_processing/mag/l1d/mag_l1d.py

```diff
@@ -122,12 +134,43 @@ def mag_l1d(
             spin_offsets=l1d_norm.spin_offsets,
             day=day,
         )
+
+        # TODO: frame specific attributes may be required
         l1d_burst.rotate_frame(ValidFrames.SRF)
         burst_srf_dataset = l1d_burst.generate_dataset(attributes, day_to_process)
         l1d_burst.rotate_frame(ValidFrames.DSRF)
         burst_dsrf_dataset = l1d_burst.generate_dataset(attributes, day_to_process)
+        l1d_burst.rotate_frame(ValidFrames.GSE)
+        burst_gse_dataset = l1d_burst.generate_dataset(attributes, day_to_process)
+        l1d_burst.rotate_frame(ValidFrames.RTN)
+        burst_rtn_dataset = l1d_burst.generate_dataset(attributes, day_to_process)
         output_datasets.append(burst_srf_dataset)
         output_datasets.append(burst_dsrf_dataset)
+        output_datasets.append(burst_gse_dataset)
+        output_datasets.append(burst_rtn_dataset)
+
+    # Output ancillary files
```

Comment:
Do we need to put ancillary somewhere in the filenames below or how does it know to put these output datasets into the ancillary location. Or are we saying they are ScienceFiles still just "extra" somehow?